### ISSUE TITLE
Mismatch in reported vs resulted evaluation accuracy on ImageNet

### ISSUE THREAD

**Issue by VSehwag** [google-research/simclr#18]
On evaluating the released checkpoints, I observe that the accuracy is a bit below the reported numbers (around 1% less). For example, with ResNet50_1x, the accuracy I got 68.2 whereas the reported number is 69.1. For ResNet50_2x, the resulted accuracy is 73.4 compared to reported 74.2. The evaluation strategy if following:
1. Using the single-center crop following your code where I first center crop along each edge with proportion=0.85 and then resize to 224x224.
2. Using the single-center crop evaluation (default from PyTorch), where we resize the shortest edge to 256 and then take a center crop of 224x224. This gives 0.3-0.5% better accuracy than the former approach. 

I wonder whether you use multiple random-crop or any other evaluation strategy in your results. Or whether the released checkpoints differ from the ones used in results for Table 6 in the paper? 
Thanks.

**Comment by chentingpc** [comment]
Did you get those results with our code? You can obtain the eval result by setting `FLAGS.mode='eval'`. Alternatively, you can use the fine-tuning command in the ReadMe page, and set the `train_epoches=0`. We only use a single center crop, see here: https://github.com/google-research/simclr/blob/f3ca72f7efc085ad4abdb65f7a63459d9cfda78f/data_util.py#L442.

**Comment by VSehwag** [comment]
I am using a local evaluation script since I am not using TPUs. I also wanted to test the effect of different cropping schemes, thus preferred to write some separate code. I am doing nothing more than loading these checkpoints and evaluating them using the data loaders mentioned earlier.

```module = hub.Module("./ref_models/simclr/ResNet50_1x/hub/", trainable=False)
x = tf.compat.v1.placeholder(shape=[None, 224, 224, 3], dtype=tf.float32)
output = module(x, signature="default", as_dict=True)
logits = output["logits_sup"]
sess.run([tf.compat.v1.global_variables_initializer(), tf.compat.v1.tables_initializer()])```

**Comment by chentingpc** [comment]
Our code can work without TPU, you can do so by setting `use_tpu=False` (see the ReadMe fine-tuning script). If you're running with standalone hub module, you probably want to check if the range of inputs is bounded between [0,  1].

**Comment by chentingpc** [comment]
Closing this for now. Can be re-opened if the issue persists.
